src/components/ProgressForm/ProgressHeader.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from '@/components/ProgressForm/ProgressForm.module.scss';
+
+function ProgressHeader() {
+    return (
+        <header className={styles.header}>
+            <p>
+                You have <span className={styles.mark}> 2 missed</span> Progress
+                Updates
+            </p>
+            <p>Lets try to avoid missing updates</p>
+        </header>
+    );
+}
```

Comment:
we should conditionally render this component based on whether there is a update missed or not. `<p>Lets try to avoid missing updates</p>` can be removed if no progress updates are missed.

Reply:
Conditional rendering and form validations will come once the API contracts are fixed